src/components/pages/AddReservation.jsx

```diff
@@ -0,0 +1,95 @@
+/* eslint-disable */
+import React, { useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { addReservation } from '../../redux/apiCalls';
+import { useNavigate } from 'react-router-dom';
+import './styles/addvehicles.css';
+
+const AddReservation = () => {
+    const [city, setCity] = useState('')
+    const [endDate, setEnddate] = useState('')
+    const [startDate, setStartDate] = useState('')
+    const [motorcycles, setMotorcycle] = useState('')
+    const [dateErr, setDateErr] = useState(null)
+    const [cityErr, setCityErr] = useState(null)
+    const [motorErr, setMotorErr] = useState(null)
```

Comment:
[OPTIONAL]
Use destructuring to extract the properties you need from the state objects - Instead of using useState for each state property and then accessing them individually, you can use destructuring to extract the properties from the state object.